### PR TITLE
Name threads used by LocalCacheManager

### PR DESCRIPTION
**Pull request by findepi**
### What changes are proposed in this pull request?

Using name threads is generally a good practice and helps system administrators understand what's going on with the system. Without explicit naming, the JDK-provided pools jave threads named `pool-N-thread-T`.

### CODE REVIEW

**Comment by findepi** [comment]
CLA signed & sent.

File: dora/core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java [hunk 1]
```diff
@@ -174,7 +178,8 @@ public static LocalCacheManager create(CacheManagerOptions options,
         }
       };
       mPagePredicate = Optional.of(ttl);
-      mTtlEnforcerExecutor = Optional.of(newScheduledThreadPool(1));
+      mTtlEnforcerExecutor = Optional.of(newScheduledThreadPool(1,
+              ThreadFactoryUtils.build("alluxio-enforcer-executor", true)));
```
Comment (jja725) [inline dora/core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java:182]: alluxio-ttl-executor

**Review by jja725 — COMMENTED** [review]
Mostly LGTM except one naming, thanks for the code quality improvement!

File: dora/core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java (hunk 1 above)
Comment (findepi) [reply dora/core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java]: thanks, fixed!

**Comment by findepi** [comment]
the build did not run, probably because it's my first time here.

**Review by jja725 — APPROVED** [review]
LGTM

**Comment by findepi** [comment]
thanks @jja725 for your review and allowing the build to run too

**Comment by jja725** [comment]
alluxio-bot, merge this please

**Comment by findepi** [comment]
@jja725 thank you for quick review & merge!

**Comment by findepi** [comment]
@jja725 do you have plans for Alluxio 313 release?

**Pull request merged**